src/main/resources/db/changelog/changes/08-create_cart_items_table.yaml

```
@@ -0,0 +1,40 @@
+databaseChangeLog:
+  - changeSet:
+      id: create_cart_items_table
+      author: DEz
```

```suggestion
      author: Volodymyr Ozemblovskyi
```
it's better to write full name of changeset's author